plugin/skills/commit-history/EXAMPLES.md

```
@@ -0,0 +1,71 @@
+```json
+{ "commits": [] }
+```
+
+Present:
+
+> No agent-linked commits on `release-2.0`. Drop the branch filter to see all
+> linked commits, or try a different branch.
+
+REST fallback for this same call, with encoding:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)